src/pepr/operator/controllers/network/policies.ts

```diff
@@ -133,12 +133,7 @@ export async function networkPolicies(pkg: UDSPackage, namespace: string, istioM
         app: `${gateway}-ingressgateway`,
       },
       port: policyPort,
-      description: getGatewayPolicyDescription(
-        policyPort!,
-        matchingClient?.clientId || undefined,
-        gateway,
-        !!waypointName,
-      ),
+      description: getGatewayPolicyDescription(serviceName, policyPort!, gateway, !!waypointName),
```

Comment:
Previously (pre-ambient authservice change) this was:
```ts
      // Use the port, selector, and gateway to generate a description for VirtualService derived policies
      description: `${policyPort}-${Object.values(selector)} Istio ${gateway} gateway`,
```

I'm not sure I'm seeing the value of `getGatewayPolicyDescription` considering it just appends `(ambient)` if in ambient mode? If ambient mode is set, but this isn't an authservice protected service that is kind of irrelevant? I might lean towards just going back to what this was before since we knew that was unique and won't cause any further changes to the naming.